servers/dsub/jobs/controllers/aggregation_controller.py

```diff
@@ -73,28 +75,39 @@ def _to_user_aggregation_entry(user_summary):
             AggregationEntry(
                 label=user_id, status_counts=StatusCounts(countsList)))
 
-    return Aggregation(name='UserId', key='userId', entries=entries)
+    return Aggregation(name='User Id', key='userId', entries=entries)
 
 
-def _get_stub_data(summaryCounts, user_aggregation):
-    # temporary fake data for testing front-end.
-    entry1 = AggregationEntry(
-        label='labelValue1',
-        status_counts=StatusCounts(counts=[
-            StatusCount(status='Succeeded', count=2),
-            StatusCount(status='Failed', count=1)
-        ]))
+def _count_job_name_summary(job, job_name_summary):
+    job_name = job.name
+    status = job.status
+
+    user_count = job_name_summary.get(job_name, {})
+
+    if status not in user_count:
+        user_count[status] = 0
+    user_count[status] += 1
+
+    job_name_summary[job_name] = user_count
+
+
+def _to_job_name_aggregation_entry(job_name_summary):
+    entries = []
+
+    for user_id, countsDict in job_name_summary.items():
+        countsList = []
+
+        for status, count in countsDict.items():
+            countsList.append(StatusCount(status=status, count=count))
+
+        entries.append(
+            AggregationEntry(
+                label=user_id, status_counts=StatusCounts(countsList)))
 
-    entry2 = AggregationEntry(
-        label='labelValue2',
-        status_counts=StatusCounts(counts=[
-            StatusCount(status='Succeeded', count=4),
-            StatusCount(status='Failed', count=6)
-        ]))
+    return Aggregation(name='Job Name', key='name', entries=entries)
 
-    ownerAggregation = Aggregation(
-        name='AnotherLabel', key='job-id', entries=[entry1, entry2])
 
+def _get_stub_data(summaryCounts, user_aggregation, job_name_aggregation):
```

Comment:
This is no longer stub data, right? Also I would inline this method, it's simple enough to not justify using a helper function.